app/public/templates/scene.html

```diff
@@ -93,10 +101,29 @@
           {% end %}
           fov="80"
           {% if scene_dict["camera_properties"]["look_controls"] %}
-            look-controls
+            {% if cursor_mode == 2 %}
+              {% if cur_scene_idx == 0 %}
```

Comment:
Why do we need to do this for the first scene? Reason I ask is that we don't do this check in the old code, so just wondering what edge case this is testing.

Reply:
There is some bug (maybe in aframe or look-controls). Without this if i just put `look-controls:...` then it hides the mouse cursor in the intro popup even though it should not hide it.

Reply:
Huh strange. @jaydhulia any thoughts?

Reply:
Explored this and basically, it is "expected" behaviour. When you have multiple cameras, one even if you switch the active camera, what you do in the other camera affects the first camera, unless you remove the relevant properties.
For example, if you go to our current deployment, open any puzzle and press any directional keys, once you exit out, you will see the room camera actually moved. 
This can be fixed by removing the appropriate property when switching camera and then adding it back, but this is super low priority bug I would say. So we should be okay doing it this way for now.